inc/helpers/namespace.php

```diff
@@ -706,14 +706,11 @@ function is_buckram() {
  */
 function get_h5p_activities( $per_page = 20 ) {
 	global $wpdb;
-	$cache_key = 'h5p_book_activities';
 
-	$page = isset( $_GET['hpage'] ) ? abs( (int) $_GET['hpage'] ) : 1;
-	if ( $page > 1 ) {
-		$offset = $page * $per_page - $per_page;
-	} else {
-		$offset = $page;
-	}
+	$page = abs( (int) ( $_GET['hpage'] ?? 1 ) );
+	$page = max( $page, 1 );
+
+	$offset = ($page - 1) * $per_page;
```

Comment:
Improved `$page` variable so id doesn't allow `page = 0`, and fixed `$offset` to correctly start at offset 0 and increment offset based on the `$page`.

Offset will be like below:
```php
$page 1 -> $offset 0
$page 2 -> $offset 20
$page 3 -> $offset 40
```